### PR TITLE
Implement integrating with database example

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Add the sample for https://github.com/wso2/ballerina-integrator/issues/6

## Approach
* Create the services required to perform CRUD operations to expose MySQL database as a service.

### CODE REVIEW

File: examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/db-interaction/data_service.bal [hunk 1]
```diff
@@ -0,0 +1,201 @@
+import ballerina/io;
```
Comment (Ibaqu) [inline examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/db-interaction/data_service.bal]: Is ballerina/io required?

Comment (Nipunich) [reply examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/db-interaction/data_service.bal]: Fixed

File: examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+target/
```
Comment (Asitha) [inline examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/.gitignore]: Shall we remove this file?

File: examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/db-interaction/data_service.bal [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+                    var res = handleUpdate(caller, msg, statusCode, payload);
+                }
+            } else {
+                json msg = "Update Failed while query execution";
+                int statusCode = 400;
+
+                var res = handleUpdate(caller, msg, statusCode, payload);
+            }      
+        } else {
+            log:printError("Payload type does not match");
```
Comment (Asitha) [inline examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/db-interaction/data_service.bal]: Shall we move error handling logic to a separate function?

File: examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/.gitignore (hunk 2 above)
Comment (Nipunich) [reply examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/.gitignore]: Fixed

File: examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/db-interaction/data_service.bal (hunk 3 above)
Comment (Nipunich) [reply examples/integration-tutorials/sending-a-simple-message-to-a-datasource/sql-data-service/db-interaction/data_service.bal]: Fixed

**Pull request merged**